sys/Makefile.dep

```diff
@@ -972,6 +971,7 @@ ifneq (,$(filter fido2_ctap,$(USEMODULE)))
   USEMODULE += crypto_aes_256
   USEMODULE += hashes
   USEMODULE += fido2
+  USEMODULE += fmt
```

Comment:
More a matter of curiosity: Was the pre-existing use of fmt_hex_bytes never reached in built examples? I don't see new code that uses fmt, and ztimer64_msec wouldn't have pulled it in either.

(That doesn't make it wrong to add it, I'm just stumped this passed before.).

Reply:
I was reached. Beforehand the FIDO2 code only ran with the CTAP USB layer also compiled which pulled in fmt and ztimer.